.ci/openshift-ci/build_install.sh

```diff
@@ -19,12 +19,68 @@ source /etc/os-release
 
 source "${cidir}/lib.sh"
 
-[ -z "$1" ] && die "Usage: $0 path/to/install/dir"
+usage() {
+	cat <<-EOF
+	Usage: $0 path/to/install/dir
+
+	This script is used by the CI job on OpenShift CI to build and install
+	Kata Containers in a given directory.
+
+	Environment variables:
+
+	- Use SANDBOXED_CONTAINERS_CONF="yes" to configure like the OpenShift
+	  Sandboxed Containers.
+	EOF
+}
+
+if [ -z "$1" ]; then
+	usage
+	die "$0: missing parameter"
+fi
+
 export DESTDIR="$1"
 info "Build and install Kata Containers at ${DESTDIR}"
 
 [ "$(id -u)" -ne 0 ] && die "$0 must be executed by privileged user"
 
+# This applies a set of build configurations akin to OpenShift
+# Sandboxed Containers.
+#
+# The following environment variables have the origin on the Fedora rawhide
+# specfile [*].
+#
+# [*] https://src.fedoraproject.org/rpms/kata-containers/blob/rawhide/f/kata-containers.spec
+sandboxed_containers_build_configs() {
+	export KERNELTYPE="compressed"
+	export DEFSHAREDFS="virtio-fs"
+	export DEFVIRTIOFSCACHESIZE=0
+	export DEFSANDBOXCGROUPONLY=true
+	export MACHINETYPE="q35"
+	export FEATURE_SELINUX="yes"
+	export DEFENABLEANNOTATIONS=['\".*\"']
+}
+
+# This ensures the guest image is built to work with the host Kernel akin to
+# OpenShift Sandboxed Containers.
+sandboxed_containers_image_configs() {
+	# It needs to overwrite some variables of
+	# kata-containers/tools/osbuilder/Makefile.
+	export DRACUT_KVERSION="4.18.0-305.19.1.el8_4.x86_64"
```

Comment:
Isn't this more fitting for `versions.yaml`?

Reply:
Good idea @Jakob-Naucke ! I've been trying to not spill OpenShift CI specific code out of the `.ci/openshift-ci` folder but I think a new entry on `versions.yaml` doesn't hurt :)

Reply:
Done.